### DISCUSSION TITLE
authentication exchange unsuccessful

### DISCUSSION THREAD

**Discussion by ioiofadhil** [Kong/kong#8369] [Help]
i tried to set up kong gateway with postgresql using ubuntu 18.04 bionic.
using documentation etc i got the following error :
```
Error: [PostgreSQL error] failed to retrieve PostgreSQL server_version_num: authentication exchange unsuccessful

  Run with --v (verbose) or --vv (debug) for more details
```
What am I missing?

**Comment by mayocream** [comment]
Can you provide the PostgreSQL version in use and the detail logs with `--v --vv` flags?

There is a related issue:
- https://github.com/Kong/kong/issues/8259.
